mlflow/store/artifact/gcs_artifact_repo.py

```diff
@@ -141,3 +148,109 @@ def delete_artifacts(self, artifact_path=None):
         blobs = gcs_bucket.list_blobs(prefix=f"{dest_path}")
         for blob in blobs:
             blob.delete()
+
+    @staticmethod
+    def _validate_support_mpu():
+        if not compare_package_version(
+            "google-cloud-storage", "2.12.0", ">="
```

Comment:
Does `google-cloud-storage` have `__version__`?

Reply:
iiuc, `importlib.metadata.version` is the quickest way to get the version of a package as it doesn't need to perform the import action

Reply:
I saw Flask 3.0 drop `__version__` last week and recommend `importlib.metadata.version`. Let's use it.